internal/file/file_manager_service_test.go

```diff
@@ -56,3 +60,191 @@ func TestFileManagerService_UpdateFile(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, 1, fakeFileServiceClient.UpdateFileCallCount())
 }
+
+func TestFileManagerService_ConfigApply_Add(t *testing.T) {
+	ctx := context.Background()
+	tempDir := t.TempDir()
+	addAction := mpi.File_FILE_ACTION_ADD
+
+	filePath := fmt.Sprintf("%s/nginx.conf", tempDir)
```

Comment:
Prefer `filepath.Join()`, as it automatically sets the correct path separator.
```suggestion
	filePath := filepath.Join(tempDir, "nginx.conf")
```